File: android/versioned-abis/expoview-abi42_0_0/src/main/java/abi42_0_0/expo/modules/battery/BatteryPackage.java
package abi42_0_0.expo.modules.battery;

import android.content.Context;

import java.util.Collections;
import java.util.List;

import abi42_0_0.org.unimodules.core.BasePackage;
import abi42_0_0.org.unimodules.core.ExportedModule;
import abi42_0_0.org.unimodules.core.ViewManager;

public class BatteryPackage extends BasePackage {
  @Override
  public List<ExportedModule> createExportedModules(Context context) {
    return Collections.singletonList((ExportedModule) new BatteryModule(context));
  }
}

File: android/versioned-abis/expoview-abi48_0_0/src/main/java/abi48_0_0/expo/modules/battery/BatteryPackage.kt
package abi48_0_0.expo.modules.battery

import android.content.Context
import abi48_0_0.expo.modules.core.BasePackage
import abi48_0_0.expo.modules.core.ExportedModule

class BatteryPackage : BasePackage() {
  override fun createExportedModules(context: Context): List<ExportedModule> {
    return listOf(BatteryModule(context))
  }
}
